news.js

```diff
@@ -0,0 +1,47 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+
+var Request = require("request");
+var answer = 0;
+rl.question('Select your preferred news source by inserting the corresponding number:\n1. BBC NEWS \n2. FOX NEWS \n3. AL JAZEERA \n4. CNN ', (answer) => {

```

Comment:
Let us make use of template literals to avoid making a very long string

```js
// Example of a template literal
const sampleLiteral = `This is an example of a template literal
This will appear on a new line if this string is logged out. No need of the "\n" escape character
This is another line
`
```
You can even concatenate two string using template literals e.g

```js
// given two sting
const a = `firstString`;
const b = `and secondString`;

// you can concatenate them by using the + operator
const aAndB = a + ' ' + b;

// or you can use the template literals
const aAndB = `${a} ${b}`;
```